test_pull_requests

```diff
@@ -124,12 +124,44 @@ NEEDS_REBASE_LABEL          = "needs-rebase"
 GITHUB_API_BASE_URL = 'https://api.github.com'
 GITHUB_BASE_URL = 'https://github.com'
 
+CONTENT_WAITING_IN_QUEUE = "Waiting: You are in the build queue at position:"
+CONTENT_WAITING_DETERMINING_QUEUE_POS = "Waiting: Determining build queue position"
+CONTENT_WAITING_STABLE_BUILD = "Waiting for stable build of"
+CONTENT_RUNNING = "Running"
+CONTENT_WAITING = "Waiting"
+CONTENT_EVALUATING = "Evaluating for testing"
+
+CONTENT_TO_STATE = {
+  CONTENT_EVALUATING => :evaluating,
+  CONTENT_RUNNING    => :running,
+  CONTENT_WAITING    => :waiting,
```

Comment:
The current state of this mapping is clearly broken -- either we fix the aliasing and make the mapping valid or the aliasing is desired and we need to figure out a more correct mapping to use. I'd personally prefer it if this merged with a non-broken mapping (i.e. the aliasing was removed in a pre-req PR) but that's a personal preference. I'll defer to you and @danmcp